agent/agent.go

```diff
@@ -308,6 +309,21 @@ func handle(conn io.ReadWriter, op byte) error {
 			return err
 		}
 		_, _ = fmt.Fprintf(conn, "New GC percent set to %v. Previous value was %v.\n", perc, debug.SetGCPercent(int(perc)))
+
+	case SetLogLevel:
```

Comment:
noice... is this exposed as a rest endpoint mop could hit too? i expect it is since the ziti cli is using it?

Reply:
No, this is the agent endpoint, available via unix socket. Make something of these operations available via REST probably makes sense at some point.